src/main/kotlin/org/jitsi/nlj/transform/node/incoming/VideoParser.kt

```diff
@@ -59,6 +59,7 @@ class VideoParser(
             packetInfo.packet = videoRtpPacket
             packetInfo.resetPayloadVerification()
         } ?: run {
+            // TODO(brian): Shouldn't we drop the packet here?
```

Comment:
I can't think of reason not to.

Reply:
Agreed. It could also throw an exception, since the packet should have been dropped already in `RtpParser` if the PT is not known.